{"name": "Принять заказ с несуществующим id курьера", "status": "passed", "description": "Попытка принять заказ с несуществующим id курьера должна вернуть ошибку 404", "start": 1701085493777, "stop": 1701085495181, "uuid": "b5c017cf-f239-4dbe-92b9-d671ff86a419", "historyId": "f32e520d4065adbb10dec0cb77502607", "testCaseId": "f32e520d4065adbb10dec0cb77502607", "fullName": "tests.test_courier.TestCourier#test_accept_order_with_wrong_courier_id", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier"}, {"name": "subSuite", "value": "TestCourier"}, {"name": "host", "value": "Ya"}, {"name": "thread", "value": "27764-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier"}]}